packages/ui/tsconfig.json

```diff
@@ -32,7 +32,8 @@
       "~program": ["./src/features/program"],
       "~program/*": ["./src/features/program/*"],
       "~project": ["./src/features/project"],
-      "~project/*": ["./src/features/project/*"]
+      "~project/*": ["./src/features/project/*"],
+      "plotly.js": ["../node_modules/plotly.js/lib/core.js"]
```

Comment:
Was this on porpoise or autogenerated and missed?

Reply:
https://stackoverflow.com/questions/39084438/how-to-import-plotly-js-into-typescript